modular_bandastation/automapper/code/automap_template.dm

```diff
@@ -0,0 +1,21 @@
+/datum/map_template/automap_template
+	name = "Automap Template"
+	should_place_on_top = FALSE
+	keep_cached_map = FALSE
+
+	/// Our load turf
+	var/turf/load_turf
+	/// The map for which we load on
+	var/required_map
+	/// Touches builtin map. Clears the area manually instead of blacklisting
+	var/affects_builtin_map
```

Comment:
```suggestion
	name = "Automap Template"
	should_place_on_top = FALSE
	keep_cached_map = FALSE

	/// The map for which we load on
	var/required_map
	/// Touches builtin map. Clears the area manually instead of blacklisting
	var/affects_builtin_map
	/// Our load turf
	var/turf/load_turf
```